test/vault/Vault.test.ts

```diff
@@ -317,6 +317,7 @@ describe("Vault test", () => {
                 mockedBaseAggregator.smocked.latestRoundData.will.return.with(async () => {
                     return [0, parseUnits("150", 6), 0, 0, 0]
                 })
+                await accountBalance.mockMarkPrice(baseToken.address, parseEther("150"))
```

Comment:
In this file, what's the reason that we didn't remove the mock of index price?

Reply:
No, we can't.
Mocking mark price is for account value calculation, and mocking index price is for funding payment calculation.
Add more comments to describe why we need to mock mark price and index price in tests.
Fixed in 847555cfb33bae9c6fe9c1e162a9db538458b735